src/app/create-event/page.tsx

```diff
@@ -43,7 +48,7 @@ export default function CreateEvent() {
   }
 
   // Create Event Button function
-  const handleSubmit = () => {
+  const handleSubmit = async () => {
     console.log(`Title: ${title}`)
```

Comment:
You can remove these debug lines as well.

Reply:
Removed the line and lines followed with console.log